client/stylesheets/_toCat.scss

```diff
@@ -4964,8 +4976,8 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .profileSettingsButton input {
-    font-size: 1.2rem;
-    padding: 15px 22px;
+    font-size: 1rem;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered background-color, border, border-radius, box-shadow, color, float, font-size, overflow, padding